clients/go.mod

```diff
@@ -63,3 +63,12 @@ require (
 	golang.org/x/sys v0.28.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+// See https://go.dev/ref/mod#go-mod-file-retract for more information about the retract directive.
+retract (
```

Comment:
oh I didn't even know that retraction was something that was possible - this is cool!